--- libvisual-plugins-0.4.0/configure.ac.orig	2006-03-16 17:50:41.000000000 +0100
+++ libvisual-plugins-0.4.0/configure.ac	2006-03-22 21:07:28.110735500 +0100
@@ -87,6 +87,11 @@
 DYNAMIC_LOADER_LIBS="-ldl"
 AC_SUBST([DYNAMIC_LOADER_LIBS])
 
+dnl List of plugins to build
+build_input_plugins=""
+build_actor_plugins=""
+build_morph_plugins=""
+
 dnl Libraries
 
 dnl EsounD
@@ -108,11 +113,6 @@
              http://jackit.sf.net/. The jackit input plugin won't be built])
 fi
 
-dnl List of plugins to build
-build_input_plugins=""
-build_actor_plugins=""
-build_morph_plugins=""
-
 dnl GdkPixbuf
 AC_ARG_ENABLE([gdkpixbuf-plugin],
   AS_HELP_STRING([--disable-gdkpixbuf-plugin],
@@ -127,7 +127,7 @@
   if test "$HAVE_GTK" = "yes"; then
     build_actor_plugins="$build_actor_plugins gdkpixbuf"
   else
-    AC_MSG_WARN([*** GTK+ >= ${gtk_required_version} is not found.
+    AC_MSG_WARN([*** GTK+ >= gtk_required_version is not found.
 	  	The libvisual GdkPixbuf image loader plugin won't be build.
 		GdkPixbuf is included within gtk-2.0 and newer, which can be
 		downloaded at http://www.gtk.org/])
@@ -148,7 +148,7 @@
   if test "$HAVE_GSTREAMER" = "yes"; then
     build_actor_plugins="$build_actor_plugins gstreamer"
   else
-    AC_MSG_WARN([*** GStreamer >= ${gst_required_version} is not found.
+    AC_MSG_WARN([*** GStreamer >= gst_required_version is not found.
 	  	The libvisual GStreamer viewer plugin won't be build.
 		GStreamer can be downloaded from http://www.gstreamer.org/])
   fi
@@ -206,7 +206,9 @@
   [ENABLE_ALSA=yes])
 
 if test "$ENABLE_ALSA" = "yes"; then
+  save_LIBS="$LIBS"
   AM_PATH_ALSA([$LV_ALSA_MIN_VERSION], [HAVE_ALSA=yes], [HAVE_ALSA=no])
+  LIBS="$save_LIBS"
 
   if test "$HAVE_ALSA" = "yes"; then
     build_input_plugins="$build_input_plugins alsa"
@@ -363,7 +363,7 @@
   [ENABLE_ANALYZER=$enableval],
   [ENABLE_ANALYZER=yes])
 
-if test "$ENABLE_ANALYZER" = xyes; then
+if test "$ENABLE_ANALYZER" = "yes"; then
   build_actor_plugins="$build_actor_plugins lv_analyzer"
 fi
 
--- libvisual-plugins-0.4.0/plugins/actor/nastyfft/Makefile.am.orig	2006-02-24 12:24:50.000000000 +0100
+++ libvisual-plugins-0.4.0/plugins/actor/nastyfft/Makefile.am	2006-03-22 22:12:40.923271000 +0100
@@ -4,5 +4,5 @@
 
 actor_nastyfft_la_CFLAGS = -I$(top_srcdir) $(LIBVISUAL_CFLAGS) $(X_CFLAGS)
 actor_nastyfft_la_LDFLAGS = -module -avoid-version 
-actor_nastyfft_la_LIBS = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU
+actor_nastyfft_la_LIBADD = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU
 actor_nastyfft_la_SOURCES = actor_nastyfft.c
--- libvisual-plugins-0.4.0/plugins/actor/madspin/Makefile.am.orig	2006-02-24 12:24:49.000000000 +0100
+++ libvisual-plugins-0.4.0/plugins/actor/madspin/Makefile.am	2006-03-22 22:12:53.492056500 +0100
@@ -10,5 +10,5 @@
 
 actor_madspin_la_CFLAGS = -I$(top_srcdir) $(LIBVISUAL_CFLAGS) -DSTAR_DIR=\""$(imagesdir)/"\"
 actor_madspin_la_LDFLAGS = -module -avoid-version 
-actor_madspin_la_LIBS = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL 
+actor_madspin_la_LIBADD = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL 
 actor_madspin_la_SOURCES = madspin.c
--- libvisual-plugins-0.4.0/plugins/actor/pseudotoad_flower/Makefile.am.orig	2006-03-08 14:06:58.000000000 +0100
+++ libvisual-plugins-0.4.0/plugins/actor/pseudotoad_flower/Makefile.am	2006-03-22 22:12:15.729696500 +0100
@@ -4,7 +4,7 @@
 
 actor_flower_la_LDFLAGS = -module -avoid-version 
 actor_flower_la_CFLAGS = -I$(top_srcdir) $(LIBVISUAL_CFLAGS) $(X_CFLAGS)
-actor_flower_la_LIBS = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU 
+actor_flower_la_LIBADD = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU 
 
 actor_flower_la_SOURCES = \
 	actor_flower.c \
--- libvisual-plugins-0.4.0/plugins/actor/lv_gltest/Makefile.am.orig	2006-02-24 12:24:49.000000000 +0100
+++ libvisual-plugins-0.4.0/plugins/actor/lv_gltest/Makefile.am	2006-03-22 22:13:08.404988500 +0100
@@ -4,5 +4,5 @@
 
 actor_lv_gltest_la_CFLAGS = -I$(top_srcdir) $(LIBVISUAL_CFLAGS) $(X_CFLAGS)
 actor_lv_gltest_la_LDFLAGS = -module -avoid-version 
-actor_lv_gltest_la_LIBS = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU 
+actor_lv_gltest_la_LIBADD = $(LIBVISUAL_LIBS) $(X_LIBS) -lGL -lGLU 
 actor_lv_gltest_la_SOURCES = actor_lv_gltest.c
